src/pages/RFOX/components/Overview/StakingInfo.tsx

```diff
@@ -153,17 +153,38 @@ export const StakingInfo: React.FC<StakingInfoProps> = ({
           </Flex>
         </Flex>
 
-        <Card width='100%' maxWidth='400px'>
+        <Card width='100%'>
           <CardBody py={4} px={4}>
-            <Flex alignItems='center' justifyContent='space-between'>
-              <Box width='100%'>
-                <HelperTooltip label={translate('RFOX.pendingRewardsBalanceHelper')}>
-                  <Text
-                    fontSize='sm'
-                    color='text.subtle'
-                    translation='RFOX.pendingRewardsBalance'
-                  />
-                </HelperTooltip>
+            <Flex flexDir='column' gap={4}>
+              <Box>
+                <Flex flexDir='column' gap={2}>
```

Comment:
Those column values are probably wrong too as we would want them to be rows on desktop and maybe column on mobile if it doesn't fit, verify accordingly if we can keep them rows or we need to column them!